doc/02-Installation.md

```diff
@@ -15,259 +15,14 @@ Before installing Icinga DB Web, make sure you have installed the
-<!-- {% endif %} -->
-<!-- {% if centos %} -->
-[install the web server on CentOS](https://icinga.com/docs/icinga-web-2/latest/doc/02-Installation/03-CentOS/#install-the-web-server),
-<!-- {% endif %} -->
-<!-- {% if debian %} -->
-[install the web server on Debian](https://icinga.com/docs/icinga-web-2/latest/doc/02-Installation/01-Debian/#install-the-web-server),
-<!-- {% endif %} -->
-<!-- {% if rhel %} -->
-[install the web server on RHEL](https://icinga.com/docs/icinga-web-2/latest/doc/02-Installation/04-RHEL/#install-the-web-server),
-<!-- {% endif %} -->
-<!-- {% if sles %} -->
-[install the web server on SLES](https://icinga.com/docs/icinga-web-2/latest/doc/02-Installation/05-SLES/#install-the-web-server),
-<!-- {% endif %} -->
-<!-- {% if ubuntu %} -->
-[install the web server on Ubuntu](https://icinga.com/docs/icinga-web-2/latest/doc/02-Installation/02-Ubuntu/#install-the-web-server),
-<!-- {% endif %} -->
-which will then take you to the web-based setup wizard, which also covers the configuration of Icinga DB Web.
-<!-- {% endif %} --><!-- {# end if not from_source #} -->
-
-If Icinga Web has been installed but not yet set up,
-please visit Icinga Web and follow the web-based setup wizard.
-For Icinga Web setups already running,
-log in to Icinga Web with a privileged user and follow the steps below to configure the Icinga DB Web module:
-
-1. Create a new resource for the Icinga DB database via the `Configuration → Application → Resources` menu.
-2. Configure the resource you just created as the database connection for the Icinga DB Web module using the
-   `Configuration → Modules → icingadb → Database` menu.
-3. Configure the connection to the Redis server through the `Configuration → Modules → icingadb → Redis` menu.
-4. In order to acknowledge problems, force checks, schedule downtimes, etc., 
-   Icinga DB Web needs access to the Icinga 2 API.
-   For this you need an `ApiUser` object with appropriate permissions on the Icinga 2 side:
-   
-!!! tip
-
-    For single-node setups it is recommended to manage API credentials in the
-    `/etc/icinga2/conf.d/api-users.conf` file. If you are running a high-availability Icinga 2 setup,
-    please manage the credentials in the master zone.
-
-1. Please add the following Icinga 2 configuration and change the password accordingly:
-   ```
-   object ApiUser "icingadb-web" {
-       password = "CHANGEME"
-       permissions = [ "actions/*", "objects/modify/*", "objects/query/*", "status/query" ]
-   }
-   ```
-2. Restart Icinga 2 for these changes to take effect. 
-3. Then configure a command transport for Icinga DB Web
-   using the credentials you just created via the `Configuration → Modules → icingadb → Command Transports` menu.
 
-If you have previously used the monitoring module, there is an option to [migrate](10-Migration.md) some settings.
+This concludes the installation. Now proceed with the [Configuration](03-Configuration.md).
```

Comment:
Why is it that the first letter of `installation` is lowercase and the one of `Configuration` uppercase?

Reply:
There's nothing special with installation but the later one is a link I guess.